tests/test_concrete_not_packed_elf64.py

```diff
@@ -1,15 +1,15 @@
-import angr
-import claripy
-import nose
 import os
+import unittest
 import subprocess
-import logging
+import angr
+import claripy
+
 
 try:
```

Comment:
Let's make this a conditional import like this example: https://github.com/angr/archinfo/blob/master/archinfo/arch.py#L16

Then, use the `unittest.skipUnless` decorator on the test class or method instead of raising this on import